### DISCUSSION TITLE
Trivia still not available after get_movie update

### DISCUSSION THREAD

**Discussion by alipphardt** [cinemagoer/cinemagoer#466] [Q&A]
Attempting to get some trivia code back up and running after the most recent update. Here's what I'm using:

`from imdb import Cinemagoer`
`ia = Cinemagoer()`
`movie = ia.get_movie_trivia('0103644')`
`ia.update(movie, info=['trivia']` 

When running 

`movie.current_info`

I see

`['main', 'plot', 'synopsis', 'trivia']`

But get a key error when I try to access the trivia

`movie['trivia']`
`KeyError: 'trivia'`

**Comment by alberanid** [comment]
This discussion is no longer relevant due to issue https://github.com/cinemagoer/cinemagoer/issues/557: as of June 2026 Cinemagoer no longer analyzes web pages and can only rely on non-commercial datasets.
